content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -311,19 +383,17 @@ and installing the sidecar injector webhook configuration on the remote cluster
             protocol: https
             name: https-XDS
           tls:
-            mode: SIMPLE
-            credentialName: $SSL_SECRET_NAME
+            mode: PASSTHROUGH
```

Comment:
We don't want to change the config to use passthrough in the official doc. Just mention (in a tip) that you can change it. Btw, that's exactly what the test program does with a couple of sed commands. I'm actually surprised that the test isn't broken with your changed snip.

Reply:
ok. will mention it in a tip.

Reply:
PASSTHROUGH should work in more cases than the TLS simple mode.